packages/docs/src/components/header/header.tsx

```diff
@@ -48,37 +42,24 @@ export const Header = component$(() => {
         </button>
         <ul className="md:grow md:flex md:justify-end md:p-4 menu-toolkit">
           <li>
-            <a
-              href="/docs/overview/"
-              class={{ active: pathname.startsWith('/docs') }}
-              onClick$={closeMenu}
-            >
+            <a href="/docs/overview/" class={{ active: pathname.startsWith('/docs') }}>
```

Comment:
Is the menu supposed to stay open after clicking now? Seems like that should be a separate commit.

Reply:
it's a full page reload, no need to run JS, it's just doing nothing